hazelcast/src/test/java/com/hazelcast/map/DynamicMapConfigTest.java

```diff
@@ -99,27 +100,9 @@ private void updateMapConfig(String mapName, HazelcastInstance node) {
         executeOperation(node, updateMapConfigOperation);
     }
 
-    private boolean isRecordStoreExpirable(IMap map) {
-        MapProxyImpl mapProxy = (MapProxyImpl) map;
-        MapService mapService = (MapService) mapProxy.getService();
-        MapServiceContext mapServiceContext = mapService.getMapServiceContext();
-        PartitionContainer container = mapServiceContext.getPartitionContainer(0);
-        RecordStore recordStore = container.getExistingRecordStore(map.getName());
-        return requireNonNull(recordStore).isExpirable();
-    }
-
-    private boolean isEvictionEnabled(IMap map) {
```

Comment:
what about eviction listener? To have identical test with previous one, i think we also need it.